docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+* or the **Hands-on** path for a more DIY approach to run your project.
 
-::: warning
-In the 3.1.5 the **Roles & Permissions** section has been migrated into the **Settings** section.
-:::
+::::: tabs type:card
+<!-- we need 5 colons or it will conflict with the callouts markup -->
+
+:::: tab Starters
+
+## 🚀 Part 1: Create a new project with Strapi starters
 
-## 1. Install Strapi and Create a new project
+Strapi [starters](https://strapi.io/starters) are the fastest way to kickstart your project. We offer starters for various use cases (blog, e-commerce solution, corporate website, portfolio) and technologies (Gatsby, Gridsome, Next, Nuxt).
 
-:::: tabs
+From now on, in this whole guide, we will use the Gatsby blog starter as an example.
 
-::: tab yarn
+### Step 1: Run the installation script
+
+To create a [Gatsby](https://www.gatsbyjs.com/) blog using Strapi, run the following command in a terminal:
+
+<code-group>
+<code-block title="NPM">
 
 ```bash
-yarn create strapi-app my-project --quickstart
+npx create-strapi-starter my-project gatsby-blog
 ```
 
+</code-block>
+<code-block title="YARN">
+
+```bash
+yarn create strapi-starter my-project gatsby-blog
+```
+
+</code-block>
+</code-group>
+
+When terminal asks `Choose your installation type`, press Enter to select the default `Quickstart (recommended)` option, and let the magic happen!
+
+### Step 2: Register & have a look at your blog
+
+Once the installation is complete, your browser automatically opens 2 tabs.
+
+The first tab ([http://localhost:1337/admin/auth/register-admin](http://localhost:1337/admin/auth/register-admin)) is the Admin Panel, it's for the backend of your app.
+
+Complete the form to create the first Administrator user:
+
+![Registration screen in admin panel](../assets/quick-start-guide/qsg-starters-part1-01-register.png)
+
+The second tab ([http://localhost:8000](http://localhost:8000)) is for the frontend of your app, and you can already see the Gatsby blog in action:
+
+![Gatsby blog front end](../assets/quick-start-guide/qsg-starters-part1-01-gatsby_fe.png)
+
+:::warning CONGRATULATIONS! 🥳 
+Your blog is ready! You can start playing with Strapi and discover the product by yourself using our [User Guide](/user-docs/latest/getting-started/introduction.html), or proceed to part 2 below.
+
+Writing a blog is not your cup of tea? You can leave this guide and play with other [Starters](https://strapi.io/starters) on your own.
+:::
+
+## 🎨 Part 2: Play with your content
+
+Strapi [starters](https://strapi.io/starters) build a full stack application and a data structure for you, so you can start playing with your content faster.
+
+You already have the admin panel for your blog running at [http://localhost:1337/admin](http://localhost:1337/admin). Now what?
+
+:::tip TIP
+If the servers are not already running, in your terminal, `cd` into the `my-project` folder and run `npm run develop` (or `yarn develop`) to launch it.
```

Comment:
Thanks Jim. I used the plural because technically in this case `yarn develop` actually launches both the Strapi and Gatsby servers.

But I agree, let's keep things simple.